game/overlord/srpc.cpp

```diff
@@ -47,7 +72,267 @@ u32 Thread_Loader() {
   return 0;
 }
 
-// todo RPC_Player
+void* RPC_Player(unsigned int /*fno*/, void* data, int size) {
+  char namebuf[8];
+  char langbuf[8];
+
+  if (gSoundEnable) {
+    gFreeMem = QueryTotalFreeMemSize();
+    // if (!PollSema(gSema)) {
+    if (gMusic) {
+      if (!gMusicPause && !LookupSound(666)) {
+        Sound* music = AllocateSound();
+        if (music != nullptr) {
+          gMusicFade = 0;
+          gMusicFadeDir = 1;
+          SetMusicVol();
+          music->sound_handle = snd_PlaySoundVolPanPMPB(gMusic, 0, 0x400, -1, 0, 0);
+          music->id = 666;
+          music->is_music = 1;
+        }
+      }
+    }
+    //}
+
+    SetMusicVol();
+    Sound* unk = LookupSound(666);
+    if (unk != nullptr) {
+      snd_SetSoundVolPan(unk->sound_handle, 0x7FFFFFFF, 0);
+      snd_SetMIDIRegister(unk->sound_handle, 0, gFlava);
+    }
+
+    int n_messages = size / SRPC_MESSAGE_SIZE;
+    SoundRpcCommand* cmd = (SoundRpcCommand*)(data);
+    while (n_messages > 0) {
+      switch (cmd->command) {
+        case SoundCommand::PLAY: {
+          // spool- soundsn are vag sounds?
```

Comment:
yeah, that makes sense. It's a bit hacked together, but there are a few "sound effects" (started with `sound-play-by-name`) that seem to be VAG sounds and not SBK sounds, so I guess this is how it knows the difference.